blockexplorer/src/store/validatorsSlice/index.ts

```diff
@@ -36,18 +105,22 @@ const validatorsSlice = createSlice({
       state.validatorsLoading = Status.LOADING;
     });
     builder.addCase(loadValidators.fulfilled, (state, { payload }) => {
-      state.numberOfValidators = payload.length;
-      state.numberOfActiveValidators = payload.filter(
-        (v: NodeValidator) => v.connected
-      ).length;
-      state.percentageOfActiveValidators = parseInt(
-        (
-          (state.numberOfActiveValidators / state.numberOfValidators) *
-          100
-        ).toFixed(0)
-      );
-      state.validators = payload.map(mapToTableData);
-      state.validatorsLoading = Status.SUCCEEDED;
+      let responsePayload: any = payload;
+      if(responsePayload !== null && responsePayload !== undefined)
+      {
```

Comment:
I don't know if camino-suite has it, but it would be good to have unified formatting at this point, maybe a pipeline of some sort that will format the code? I would love @aeddaqqa or @Ayagoumi input here.